lgc/state/PipelineState.cpp

```diff
@@ -1278,6 +1281,13 @@ void PipelineState::setShaderDefaultWaveSize(ShaderStage stage) {
 
         if ((subgroupSize < waveSize) || getOptions().fullSubgroups)
           waveSize = subgroupSize;
+      } else if (checkingStage == ShaderStageCompute || checkingStage == ShaderStageMesh ||
+                 checkingStage == ShaderStageTask) {
+        // If workgroup size is not larger than 32, use wave size 32.
+        auto &mode = m_shaderModes.getComputeShaderMode();
```

Comment:
For mesh shader, should use getMeshShaderMode() to get the workgroup size. I didn't add it to compute mode because mesh shader has some graphics attributes :)

Reply:
Thanks. Updated to use 'getMeshShaderMode()' for mesh.